origin-faucet/test/lib-token.js

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert'
+import helper from './_helper'
+import Token from '../lib/token'
+
+// These tests are for the token library that the token CLI uses. We don't
```

Comment:
super nit: update comment to remove mention of the token cli. also in the test description on line 8 below.